Content.Client/SS220/Thermals/ThermalVisionSystem.cs

```diff
@@ -0,0 +1,78 @@
+// Original code github.com/CM-14 Licence MIT, EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
```

Comment:
```suggestion
// Original code github.com/CM-14 Licence MIT, All edits under © SS220, EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
```
Правильно подметили, что лицензия написана непонятно, предлагаю такое решение, хотя лучше спросить кого по опытнее.